### PR TITLE
CDPT-689 production release for php8.2 upgrade

### PR DESCRIPTION
**Pull request by wilson1000-MoJ**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,25 +1,13 @@
...
-    echo 'deb http://apt.newrelic.com/debian/ newrelic non-free' > /etc/apt/sources.list.d/newrelic.list && \
-    curl -fsSL https://download.newrelic.com/548C16BF.gpg | apt-key add - && \
-    mv docker/conf/nginx/server.conf /etc/nginx/sites-available/ && \
-    mv docker/conf/php-fpm/newrelic.ini /etc/php/7.4/fpm/conf.d/ && \
-    mv docker/init/configure-maintenance-mode.sh /etc/my_init.d/ && \
-    chmod +x /etc/my_init.d/configure-maintenance-mode.sh && \
-    apt-get update && \
-    apt-get install -y libffi-dev newrelic-php5 && \
-    rm -f /etc/my_init.d/configure-ip-whitelist.sh
+RUN mv docker/conf/nginx/server.conf /etc/nginx/sites-available/
```
Comment (digitalali-moj) [inline Dockerfile:10]: Does this mean we're not using PHP FPM now?

File: web/app/themes/clarity/src/components/c-input-container/on-change.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export default (function ($) {
```
Comment (digitalali-moj) [inline web/app/themes/clarity/src/components/c-input-container/on-change.js:1]: Just checking if this is part of the search enhancement and if it should go in with this.

**Review by digitalali-moj — APPROVED** [review]
Excellent work, and a lot! Some ignorant comments left. Would be nice to talk through bits like FPM, buildspec.yml to understand a bit better!

File: Dockerfile (hunk 1 above)
Comment (wilson1000-MoJ) [reply Dockerfile:10]: Hey Jav. All deleted lines are included in our base image; intranet-base:latest

File: Makefile [hunk 3]
```diff
@@ -18,20 +18,24 @@ docker-clean:
 	bin/docker-clean.sh
 
 # Run the application
-run:
-	docker-compose up
+run: dory
```
[the remark this answers is not in the document]
Comment (wilson1000-MoJ) [reply Makefile:21]: Not in this case mate; dory references a make command on line 33.

**Comment by wilson1000-MoJ** [comment]
Thanks so much. Let's have a catch up tomorrow.

File: web/app/themes/clarity/src/components/c-input-container/on-change.js (hunk 2 above)
Comment (wilson1000-MoJ) [reply web/app/themes/clarity/src/components/c-input-container/on-change.js:1]: Good question. 
This is to do with supporting radio elements on the search form. I've added it to compliment the form builder api which we will be using in the next couple of weeks so left it in for that. Happy to walk through what I did tomorrow, if you're free buddy.

**Pull request merged**